.github/workflows/discussionbot-workflow.yml

```diff
@@ -0,0 +1,35 @@
+name: Discussion Help Bot
+on:
+  discussion:
+    types:
+    # Use locked as trigger event until this chatbot is ready
+      - created
+    #  - locked
```

Comment:
I wonder if there are other events that should trigger this. Is there an event associated with editing the content?